project/src/main/java/com/google/sps/servlets/CandidateServlet.java

```diff
@@ -64,7 +63,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   // @TODO [Function for getting (1) official election/candidate information from the database.]
-
+  private Candidate getCandidateData(String candidateId) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query candidateQuery = new Query("Candidate")
+        .setFilter(new FilterPredicate("candidateId", FilterOperator.EQUAL,
+                   KeyFactory.createKey("Candidate", Long.parseLong(candidateId))));
+    PreparedQuery candidateQueryResult = datastore.prepare(candidateQuery);
+    Entity candidateData = candidateQueryResult.asSingleEntity();
+    return new Candidate(candidateId,
```

Comment:
Are you planning on adding some sort of blurb about the candidate at all? I think it could potentially be helpful to users, but I can imagine it might be difficult to implement with scraping.

Reply:
I am planning on adding a blurb in the next or following PR. My main focus is to get our initial data-store info displayed. I will add a todo.